internal/cli/atlas/deployments/setup.go

```diff
@@ -177,15 +177,18 @@ func (opts *SetupOpts) configureMongod(ctx context.Context, keyFileContents stri
 			Cmd: "/bin/sh",
 			Args: []string{
 				"-c",
-				`echo $KEYFILECONTENTS > $KEYFILE && \
-                   chmod 400 $KEYFILE && \
-                   python3 /usr/local/bin/docker-entrypoint.py \
-                                          --transitionToAuth \
-                                          --dbpath $DBPATH \
-                                          --keyFile $KEYFILE \
-                                          --replSet $REPLSETNAME \
-                                          --setParameter "mongotHost=$MONGOTHOST" \
-                                          --setParameter "searchIndexManagementHostAndPort=$MONGOTHOST"`,
+				`if [ ! -f $KEYFILE ]
```

Comment:
without this check, stopping and starting back mongod will fail (as keyfile is already there)

Reply:
nice!

Reply:
these file are becoming a bit gnarly, you should consider moving them to dedicates `.sh` files so they are checked with shellcheck and include them via file embedding, see https://github.com/mongodb/mongodb-atlas-cli/blob/fdb8b0cf8b9a87be29095b2d4edfc56fccdd652d/tools/chocolateypkg/chocolateypkg.go#L68C1-L69

Reply:
good point, done